recipes/ginkgo/all/conanfile.py

```diff
@@ -102,7 +117,10 @@ def loose_lt_semver(v1, v2):
 
     def build_requirements(self):
         if Version(self.version) >= "1.7.0":
-            self.tool_requires("cmake/[>=3.16 <4]")
+            if self.options.cuda:
+                self.tool_requires("cmake/[>=3.18 <4]")
```

Comment:
Any details as to why this is needed are appreciated, thanks!

Reply:
See the [README](https://github.com/ginkgo-project/ginkgo?tab=readme-ov-file#linux-and-mac-os), the CUDA build requires a more recent CMake version.